code/datums/gamemode/factions/bloodcult/bloodcult_items.dm

```diff
@@ -1,16 +1,350 @@
+		to_chat(user, "<span class='warning'>As you reach to pick up \the [src], you feel a searing heat inside of you!</span>")
+		playsound(loc, 'sound/effects/sparks2.ogg', 50, 1, 0,0,0)
+		user.Knockdown(5)
+		user.Stun(5)
+		icon_state = "tome"
+		item_state = "tome"
+		flick("tome-stun",src)
+		state = TOME_CLOSED
+		if(iscarbon(loc))
+			var/mob/living/carbon/M = loc
+			M.update_inv_hands()
+		return
+	..()
+
+/obj/item/weapon/tome/pickup(var/mob/user)
+	if(iscultist(user) && state == TOME_OPEN)
+		usr << browse_rsc('icons/tomebg.png', "tomebg.png")
+		usr << browse(tome_text(), "window=arcanetome;size=512x375")
+
+/obj/item/weapon/tome/dropped(var/mob/user)
+	usr << browse(null, "window=arcanetome")
+
+/obj/item/weapon/tome/attack_self(var/mob/living/user)
+	if(!iscultist(user))//Too dumb to live.
+		user.take_organ_damage(0,10)
+		to_chat(user, "<span class='warning'>You try to peek inside \the [src], only to feel a discharge of energy and a searing heat inside of you!</span>")
+		playsound(loc, 'sound/effects/sparks2.ogg', 50, 1, 0,0,0)
+		user.Knockdown(5)
+		user.Stun(5)
+		if (state == TOME_OPEN)
+			icon_state = "tome"
+			item_state = "tome"
+			flick("tome-stun",src)
+			state = TOME_CLOSED
+		else
+			flick("tome-stun2",src)
+		return
+	else
+		if (state == TOME_CLOSED)
+			icon_state = "tome-open"
+			item_state = "tome-open"
+			flick("tome-flickopen",src)
+			state = TOME_OPEN
+			usr << browse_rsc('icons/tomebg.png', "tomebg.png")
+			usr << browse(tome_text(), "window=arcanetome;size=537x375")
+		else
+			icon_state = "tome"
+			item_state = "tome"
+			flick("tome-flickclose",src)
+			state = TOME_CLOSED
+			usr << browse(null, "window=arcanetome")
+
+		if(iscarbon(loc))
+			var/mob/living/carbon/M = loc
+			M.update_inv_hands()
+
+//absolutely no use except letting cultists know that you're here.
+/obj/item/weapon/tome/attack_ghost(var/mob/dead/observer/user)
+	if (state == TOME_OPEN && can_flick)
+		if (Adjacent(user))
+			to_chat(user, "You flick a page.")
+			flick("tome-flick",src)
+			can_flick = 0
+			spawn(5)
+				can_flick = 1
+		else
+			to_chat(user, "<span class='warning'>You need to get closer to interact with the pages.</span>")
+
+/obj/item/weapon/tome/attackby(var/obj/item/I, var/mob/user)
+	if (..())
+		return
+	if (istype(I, /obj/item/weapon/talisman))
+		if (talismans.len < MAX_TALISMAN_PER_TOME)
+			if(user.drop_item(I))
+				talismans.Add(I)
+				I.forceMove(src)
+				to_chat(user, "<span class='notice'>You slip \the [I] into \the [src].</span>")
+				if (state == TOME_OPEN)
+					usr << browse_rsc('icons/tomebg.png', "tomebg.png")
+					usr << browse(tome_text(), "window=arcanetome;size=512x375")
+		else
+			to_chat(user, "<span class='warning'>This tome cannot contain any more talismans. Use or remove some first.</span>")
+
+#undef PAGE_FOREWORD
+#undef PAGE_LORE1
+
+///////////////////////////////////////TALISMAN////////////////////////////////////////////////
+
+/obj/item/weapon/talisman
```

Comment:
Something to think about:

PROS
* will not show up on the paper items list when spawning paper

CONS
* won't fit in places paper will go (clipboards, paperbins, etc)
* loses some paperlike behaviors like lighting with a lighter and wiping clean using ethanol

Reply:
Tomes are now the alternative to store talismans. One that makes a bit more sense.
Also most paperlike behaviors caused bugs when I tried to perform them on the server, such as trying to write on one.

Reply:
This also means that future updates to papers won't break talismans.